library/std/src/sync/condvar.rs

```diff
@@ -21,11 +21,11 @@ impl WaitTimeoutResult {
     ///
     /// # Examples
     ///
-    /// This example spawns a thread which will update the boolean value and
-    /// then wait 100 milliseconds before notifying the condvar.
+    /// This example spawns a thread which will sleep 20 milliseconds before
+    /// updating the condvar and then notify.
```

Comment:
Where you're going seems like a good direction but currently it ends up at... "and then notify"... notify what?